### PR TITLE
Epiap accessibility path link enhancement (in next)

### PR DESCRIPTION
**Pull request by ue71603**
added french specialities

replaces #538

### CODE REVIEW

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 1]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+		</xsd:restriction>
+	</xsd:simpleType>
+	<xsd:simpleType name="TactileGuidingStripStatusEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the status of the Tactile Guiding Strip</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="goodAndContrasted">
+				<xsd:annotation>
+					<xsd:documentation>when there guiding strip has the expected quality level</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:341]:
```suggestion
					<xsd:documentation>When there guiding strip has the expected quality level.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 2]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="primitive">
+				<xsd:annotation>
+					<xsd:documentation>when there is no guide strip as such but a decorative element or ledge can be used for this purpose</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:346]:
```suggestion
					<xsd:documentation>When there is no guide strip as such but a decorative element or ledge can be used for this purpose.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 3]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="incorrect">
+				<xsd:annotation>
+					<xsd:documentation>when there is a guide strip but it does not correspond to a path that can actually be used (passes through the middle of an obstacle without going around it for example)</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:351]:
```suggestion
					<xsd:documentation>When there is a guide strip but it does not correspond to a path that can actually be used (passes through the middle of an obstacle without going around it for example).</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 4]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="bad">
+				<xsd:annotation>
+					<xsd:documentation>when the guide strip is degraded and difficult to interpret</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:356]:
```suggestion
					<xsd:documentation>When the guide strip is degraded and difficult to interpret.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd [hunk 5]
```diff
@@ -186,11 +191,21 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Nature of the  tactile warning strips (in the direction of the PATH LINK). +v1.1</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="TactileWarningStripContrast" type="xsd:boolean" minOccurs="0" default="true">
+				<xsd:annotation>
+					<xsd:documentation>indicates whether the color of the possible warning strip is sufficiently contrasted with the color of the floor. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:196]: Remove OSM reference.

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd [hunk 6]
```diff
@@ -186,11 +191,21 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:element>
 			<xsd:element name="TactileGuidingStrip" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
 					<xsd:documentation>Indicates whether there are guiding strips. +v1.1</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="TactileGuidingStripStatus" type="TactileGuidingStripStatusEnumeration" minOccurs="0" default="goodAndContrasted">
+				<xsd:annotation>
+					<xsd:documentation>Provides additional details on possible guide strip.  To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]: Remove OSM reference.

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 7]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+		</xsd:restriction>
+	</xsd:simpleType>
+	<xsd:simpleType name="TactileGuidingStripStatusEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the status of the Tactile Guiding Strip</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="goodAndContrasted">
+				<xsd:annotation>
+					<xsd:documentation>When there guiding strip has the expected quality level.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:341]:
```suggestion
					<xsd:documentation>When the guiding strip has the expected quality level.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 8]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="primitive">
+				<xsd:annotation>
+					<xsd:documentation>When there is no guide strip as such but a decorative element or ledge can be used for this purpose.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:346]:
```suggestion
					<xsd:documentation>When there is no guiding strip as such but a decorative element or ledge can be used for this purpose.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 9]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="incorrect">
+				<xsd:annotation>
+					<xsd:documentation>When there is a guide strip but it does not correspond to a path that can actually be used (passes through the middle of an obstacle without going around it for example).</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:351]:
```suggestion
					<xsd:documentation>When there is a guiding strip but it does not correspond to a path that can actually be used (passes through the middle of an obstacle without going around it for example).</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd [hunk 10]
```diff
@@ -319,6 +319,45 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="bad">
+				<xsd:annotation>
+					<xsd:documentation>When the guide strip is degraded and difficult to interpret.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_support.xsd:356]:
```suggestion
					<xsd:documentation>When the guiding strip is degraded and difficult to interpret.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd [hunk 11]
```diff
@@ -171,6 +171,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Type of flooring of the walking surface. +v1.1</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="FlooringStatus" type="FlooringStatusEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Status of the flooring</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:176]:
```suggestion
					<xsd:documentation>Status of the flooring.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd (hunk 5 above)
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:196]:
```suggestion
					<xsd:documentation>Indicates whether the color of the possible warning strip is sufficiently contrasted with the color of the floor. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd (hunk 6 above)
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:206]:
```suggestion
					<xsd:documentation>Provides additional details on possible guiding strip. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd (hunk 5 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd]: done

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd (hunk 6 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd]: done

**Comment by ue71603** [comment]
@trurlurl @skinkie updated

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd [hunk 12]
```diff
@@ -193,7 +193,7 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="TactileWarningStripContrast" type="xsd:boolean" minOccurs="0" default="true">
 				<xsd:annotation>
-					<xsd:documentation>indicates whether the color of the possible warning strip is sufficiently contrasted with the color of the floor. To compare with the OpenStreetMap tag tactile_paving https://wiki.openstreetmap.org/wiki/FR:Key:tactile_paving</xsd:documentation>
+					<xsd:documentation>indicates whether the color of the possible warning strip is sufficiently contrasted with the color of the floor.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd:196]:
```suggestion
					<xsd:documentation>Indicates whether the color of the possible warning strip is sufficiently contrasted with the color of the floor.</xsd:documentation>
```

**Review by trurlurl — APPROVED** [review]

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd (hunk 6 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd]: done

Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd]: done

Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_path_version.xsd]: done

**Review by Aurige — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**